### PR TITLE
0.13.0 Changelog

### PR DESCRIPTION
**Pull request by alazier**
@appden

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+### Breaking changes
+* With this release we have switched over to a new cross platform compatible date format. This change 
+  will only require action from users who are using both the JS and Cocoa or Android bindings simutaneously and accessing
```
Comment (appden) [inline CHANGELOG.md]: `simultaneously` is misspelled

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+  Realm files from both bindings. In these cases you need to open the Realm file with the latest version of
+  the iOS or Android bindings before accessing the Realm from JS to prevent an unnecessary conversion.
+
+### Enhancements
+* None
+
+### Bugfixes
+* Date properties are now stored in a format compatible with the browser and other bindings.
```
Comment (appden) [inline CHANGELOG.md]: I would say `Realm Browser`

**Comment by appden** [comment]
:+1:

**Comment by appden** [comment]
We can leave this open for all changelog additions for the next release if you prefer

**Comment by appden** [comment]
@alazier the rest look good?

**Comment by alazier** [comment]
👍

**Pull request merged**